content/src/content/jcr_root/apps/core/wcm/components/accordion/v1/accordion/clientlibs/site/js/accordion.js

```diff
@@ -111,7 +111,7 @@
 
                 // Expand the item based on deep-link-id if it matches with any existing accordion item id
                 var deepLinkIdIndex = -1;
-                if (config.deepLinkId && config.deepLinkId.substring("-panel-") !== -1) {
+                if (config.deepLinkId && config.deepLinkId.substring("-item-item") !== -1) {
```

Comment:
Shouldn't this be `-item` rather than `-item-item`?

Reply:
@richardhand As we are adding item name at the end of the id. All of the elements, button, panel and item (the wrapper div) all has item in common.
So, to avoid any confusion I added the 'item' which is added on wrapper div and the item coming from name to make it unique.
I am adding a screen shot for how the ids look like.

![image](https://user-images.githubusercontent.com/12025474/76779221-3b74ee00-6768-11ea-8f58-feb3421319a4.png)